src/main/java/com/artipie/ArtipieException.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * The MIT License (MIT) Copyright (c) 2020-2021 artipie.com
+ * https://github.com/artipie/asto/LICENSE.txt
+ */
+package com.artipie;
+
+/**
+ * Base Artipie exception.
+ * <p>It should be used as a base exception for all Artipie public APIs
+ * as a contract instead of others.</p>
+ *
+ * @since 1.0
+ * @implNote ArtipieException is unchecked exception, but it's a good
+ *  practice to not it via {@code throws} tag in JavaDocs.
```

Comment:
@g4s8 hm, seems like there is some mistake in the sentence, I do not understand the meaning of the second part